erpnext/setup/doctype/designation/designation.json

```diff
@@ -8,6 +8,7 @@
  "document_type": "Setup",
  "engine": "InnoDB",
  "field_order": [
+  "enabled",
```

Comment:
@zeel-codder You should add a disabled code instead. Once the disabled check is enabled the designation will by default be filtered in the link fields as well

Reply:
Has done the changes @deepeshgarg007